deploy.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+# script for deploying to any environment, choosing the matching deploy password environment variable
+
+# arg 1: environment to deploy to
+env=$1
+
+# convert env to uppercase to match environment variable case
+upperenv=`echo "$env" | tr '[:lower:]' '[:upper:]'
+`
+eval "password=\$APP_CREDENTIALS_${upperenv}_PWD"
+
+git fetch --unshallow
+git push --force https://$bat-$env-manage-courses-ui-app:$password@bat-$env-manage-courses-ui-app.scm.azurewebsites.net:443/bat-$env-manage-courses-ui-app.git HEAD:refs/heads/master
```

Comment:
Does `$bat` not need escaping here? Or does `bash` pass it through if it can't find a variable named `bat`?

Reply:
please refer to master jake work...


https://github.com/DFE-Digital/manage-courses-support/pull/71/files

Reply:
if i recall correctly its in its own massive file so you can execute it firstly without need travis to see if it works

Reply:
@defong Sorry, not entirely clear on what you're saying.

@timabell Just did a quick try and you'll need to escape `$bat` with a `\`:

```bash
bash-3.2$ env=dev
bash-3.2$ echo $bat-$env
-dev
```

Reply:
got spot on the escaping - copy-paste errorror

Reply:
@defong that was superceded by newer PRs
yes you're right, should have run it :-D